mixer/cmd/config.go

```diff
@@ -74,10 +76,47 @@ variables will not be expanded and the values will not be validated`,
 	},
 }
 
+var configSetCmd = &cobra.Command{
+	Use:   "set <property> <value>",
+	Short: "Set a property in the config file to a given value",
+	Long: `This command will parse the provided property in the format 'Section.Property',
+	assign the provided value and update the config file. The command will only validate
+	the existence of the provided property, but will not validate the value provided.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) != 2 {
+			fail(errors.New("config set requires exactly 2 parameters"))
+			return
+		}
```

Comment:
yeah, I saw that on msrj`s PR after I sen't this. I'm pretty sure there are other parts of our code that perfom this check, so it might be worth double checking our older commands.